tests/job/test_external.py

```diff
@@ -0,0 +1,40 @@
+import os
+import unittest
+
+from pyiron_base import Project
+
+job_py_source = """
+from pyiron_base import Notebook as nb
+coeff_tot = nb.get_custom_dict()
+print(coeff_tot['value'])
+"""
+
+class TestScriptJob(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.file_location = os.path.dirname(os.path.abspath(__file__)).replace(
+            "\\", "/"
+        )
```

Comment:
Why do you need `replace`?

Reply:
I copied this from [GenericJob](https://github.com/pyiron/pyiron_base/blob/master/tests/job/test_genericJob.py), my guess is it related to handling of windows paths.

Reply:
Isn't it also the reason why appveyor is failing?

Reply:
Could be, will remove it.  I thought that it was needed when calling `Project`, @jan-janssen do you know if it is necessary?

Reply:
`test_scriptjob.py` has the same issue

Reply:
Ah but wait I see that `test_external.py` is already failing. So it was probably not the reason.

Reply:
The replace part is used because pyiron internally also replaces windows path separation `\\` with `/`. So if you want to compare the path then it is useful to keep it, that is why I added it as default to most tests.